lib/util/exec/containerbase.ts

```diff
@@ -174,7 +175,7 @@ const allToolConfig: Record<string, ToolConfig> = {
   pixi: {
     datasource: 'github-releases',
     packageName: 'prefix-dev/pixi',
-    versioning: pep440VersioningId,
```

Comment:
when we add this config we do not have conda versioning yet, so we can change it to conda versioning now, since we currently have a `requires-pixi` is conda versioning.

Reply:
Can this be not moved to a separate PR and merged simply before this one?

Reply:
https://github.com/renovatebot/renovate/pull/35118

Reply:
https://github.com/renovatebot/renovate/pull/35118